### PR TITLE
added link to flyers #66

### PR DESCRIPTION
**Pull request by lucadelu**

### CODE REVIEW

File: content/about/brand.md [hunk 1]
```diff
@@ -140,6 +140,21 @@ grey-color-light-bg mb-2"></div>
...
 
+### GRASS flyers
+
+<div>
+<p> For a simple overview of <b>GRASS GIS</b> you can download one of the two <b>GRASS GIS flyer</b> </p>
+<ul>
+  <li>
+      <b>OSGeo</b> one sheet flyer <a href="https://github.com/OSGeo/osgeo/raw/master/marketing/print/one-page-info/eu/grass_gis_a4.pdf" class="inl btn btn-primary" target="_blank">Download PDF</a>
+  </li>
+  <li><b>GRASS community</b> threefold flyer <a href="https://trac.osgeo.org/grass/export/74509/grass-promo/grassflyer/flyer_2016/en/grass_flyer.pdf" class="inl btn btn-primary" target="_blank">Download PDF</a>
```
Comment (neteler) [inline content/about/brand.md:151]:
```suggestion
  <li><b>GRASS community</b> threefold flyer <a href="https://trac.osgeo.org/grass/export/74509/grass-promo/grassflyer/flyer_2016/en/grass_flyer.pdf" class="inl btn btn-primary" target="_blank">Download PDF</a>
```

better use a link to https://github.com/OSGeo/grass-promo/tree/master/grassflyer ?

File: content/about/brand.md [hunk 2]
```diff
@@ -140,6 +140,21 @@ grey-color-light-bg mb-2"></div>
 </div>
 
 
+### GRASS flyers
+
+<div>
+<p> For a simple overview of <b>GRASS GIS</b> you can download one of the two <b>GRASS GIS flyer</b> </p>
```
Comment (veroandreo) [inline content/about/brand.md:146]:
```suggestion
<p> For a simple overview of <b>GRASS GIS</b> you can download one of the two <b>GRASS GIS flyers</b> </p>
```

File: content/about/brand.md [hunk 3]
```diff
@@ -140,6 +140,21 @@ grey-color-light-bg mb-2"></div>
...
+<ul>
+  <li>
+      <b>OSGeo</b> one sheet flyer <a href="https://github.com/OSGeo/osgeo/raw/master/marketing/print/one-page-info/eu/grass_gis_a4.pdf" class="inl btn btn-primary" target="_blank">Download PDF</a>
+  </li>
+  <li class="mt-20"><b>GRASS community</b> threefold flyer <a href="https://github.com/OSGeo/grass-promo/raw/master/grassflyer/flyer_2016/en/grass_flyer.pdf" class="inl btn btn-primary" target="_blank">Download PDF</a>
+      <br/>
+      Other languages are available <a href="https://trac.osgeo.org/grass/browser/grass-promo/grassflyer/flyer_2016?order=name" target="_blank">here</a>
```
Comment (veroandreo) [inline content/about/brand.md:153]:
```suggestion
      Languages other than English are available <a href="https://trac.osgeo.org/grass/browser/grass-promo/grassflyer/flyer_2016?order=name" target="_blank">here</a>
```

Comment (neteler) [reply content/about/brand.md:153]:
```suggestion
      Other languages are available <a href="https://github.com/OSGeo/grass-promo/tree/master/grassflyer/flyer_2016/" target="_blank">here</a>
```

**Pull request merged**